app/controllers/deploys_controller.rb

```diff
@@ -139,6 +134,6 @@ def find_deploy
   end
 
   def active_deploy_scope
-    @project ? @project.deploys.active : Deploy.active
+    current_project ? current_project.deploys.active : Deploy.active
```

Comment:
@jonmoter , this line has confused me a little bit. I created this branches from your initial branch jonmoter/add-project-admins. In that branch, you'll find a difference to master in the way how find_project is used:
- In master:
```
before_action :find_project
```
- In your original branch:
```
before_action :find_project, except: [:active, :active_count, :recent, :changeset]
```
So, in the way this was implemented in the branch, I believe that when the `active` and `active_count` actions were executed, `@project` was never being initialised, and we were always returning `Deploy.active`, unless I missed something. This was probably hard to notice because the fallback was to return all active deploys.

I kept your changes, because I noticed we need to be able to load all active deploys when outside the context of a project, but, in this line, I think we should first make an attempt to load the project, if `project_id` is present in the params, and fallback to list all active deploys, otherwise. That's why I used the `current_project` method.

Can you double check if I'm missing something?